File: C++/Example Looping/Uncounted Loop/Do While.cpp
#include <iostream>
using namespace std;

int main(){
	
	char vartry = 'y';
	int i = 0;
	
	do {
	cout << " " << endl;
	cout << "Enter y for Example" << endl;
	cout << " " << endl;
	cout << "Enter : ";
	cin >> vartry;
	i++;
	}while(vartry == 'y');
	
	cout << " " << endl;
	cout << "Done" << endl;

	return 0;
}

File: C#/Example Looping/Uncounted Loop/Do While.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            char vartry = 'y';
            int i = 0;

            do
            {
                Console.WriteLine();
                Console.WriteLine("Enter y for Example");
                Console.WriteLine();
                Console.Write("Enter : ");
                vartry = char.Parse(Console.ReadLine());
                i++;
            } while (vartry == 'y');

            Console.WriteLine();
            Console.WriteLine("Done");

        }
    }
}
